lib/ansible/modules/cloud/amazon/iam_cert.py

```diff
@@ -33,11 +33,13 @@
   new_name:
     description:
       - When present, this will update the name of the cert with the value passed here.
+      - When present, cert, key and cert_chain values will be ignored.
     required: false
     aliases: []
```

Comment:
Unfortunately I had to re-add one of the "required: false" lines to fix a merge conflict.